### ISSUE TITLE
training setting for single-task (i.e., semantic segmentation in ade20k) in painter

### ISSUE THREAD

**Issue by yixuan730** [baaivision/Painter#45]
Hi, what are the specific training settings for single-task of semantic segmentation in ade20k?
Now, our settings are: batch size=8 (per gpu), accumulate iterations=16, nodes=2, base lr=1e-3 (also tried actual lr=1e-3 which not worked), epoch=300, warmup epoch=20, layer decay=0.8. 
However, this hyper-params setting is not worked. Is there something we're missing?
Thank you very much!

**Comment by essunny310** [comment]
I also trained Painter for single-task of semantic segmentation in ADE20K.
My setting is: batch size=1 (per GPU), accumulate iteration=16. Other parameters are not modified. The model was trained on a single machine with 4 GPUs (RTX 3090).

- After 15 epochs (which is the default setting), the mIoU I got is 9.3
- After 100 epochs, the mIoU I got is 32.2

Hi @yixuan730, could you please report the value you got? Thanks!

Hi @WXinlong, I was wondering if I missed something or the model was sensitive to batch size. Thank you so much!

**Comment by TXH-mercury** [comment]
@essunny310  similar situation.
batch size 2 * accum 16 * cards 8,  15 epoch, lr=1e-3  miou~12.8
batch size 2 * accum 16 * cards 8,  15 epoch, lr=1e-4  miou~32
The visualization of these two settings are similar: the color in each patch are totally the same and shows no details.
I am trying with bigger batch size and bigger epoch.

**Comment by LIUTAOGE** [comment]
train adek20 没有效果，请问解决了吗，谢谢

**Comment by GZ-YourZY** [comment]
我通
<img width="282" alt="Snipaste_2023-12-12_13-26-24" src="https://github.com/baaivision/Painter/assets/77562198/b7d3ad7b-9cc9-408b-b9c5-6bfeca0237af">
我通过训练得到的pth文件并不能正确推理，而是返回这样的图片，请问这是因为什么原因

**Comment by mrwu-mac** [comment]
> @essunny310 similar situation. batch size 2 * accum 16 * cards 8, 15 epoch, lr=1e-3 miou~12.8 batch size 2 * accum 16 * cards 8, 15 epoch, lr=1e-4 miou~32 The visualization of these two settings are similar: the color in each patch are totally the same and shows no details. I am trying with bigger batch size and bigger epoch.

I got the same problem, do u have new results?

**Comment by ImmortalSdm** [comment]
> 我通 <img alt="Snipaste_2023-12-12_13-26-24" width="282" src="https://private-user-images.githubusercontent.com/77562198/289753548-b7d3ad7b-9cc9-408b-b9c5-6bfeca0237af.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzI3MDEzMjAsIm5iZiI6MTc3MjcwMTAyMCwicGF0aCI6Ii83NzU2MjE5OC8yODk3NTM1NDgtYjdkM2FkN2ItOWNjOS00MDhiLWI5YzUtNmJmZWNhMDIzN2FmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAzMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMzA1VDA4NTcwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQwZTFiNGQyNWU1ZGVjYjNjZDg0ZDlhZDAzN2M4NGRlOTY4ZWJmMzNkNTdmYTdkMDQzZjUwYzg0Mzc3ZTI4NmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.BzX65-W4CxiKTzpNDsI-xdr0Iw5mtb8_-6MsBr7Q-og"> 我通过训练得到的pth文件并不能正确推理，而是返回这样的图片，请问这是因为什么原因

Same problem, any ideas? @LIUTAOGE 

<img width="896" height="896" alt="Image" src="https://github.com/user-attachments/assets/320ca886-1c64-4a8b-a7d6-49af507766d7" />
